src/lib/slacko.ml

```diff
@@ -695,12 +690,26 @@ let process request =
 
 let (<<) f g x = f @@ g x
 
-let query =
-  process << Cohttp_unix.Client.get
+let auth_header token =
+  Cohttp.Header.init_with "Authorization" ("Bearer " ^ token)
+
+let endpoint base_url request =
+  let url = Uri.of_string (base_url ^ request.method') in
+  List.fold_left (Uri.add_query_param') url request.arguments
+
+let unauthed_query ?(base_url=default_base_url) =
+  process << Cohttp_unix.Client.get << endpoint base_url
```

Comment:
I know we use `<<` here, but I think to avoid excessive cuteness and hard to understand code, we should probably just use `|>` instead.

Reply:
Excessive cuteness 😹